src/handlers/sdp/commonUtils.ts

```diff
@@ -172,8 +172,8 @@ export function extractDtlsParameters(
 ): DtlsParameters
 {
 	const mediaObject = (sdpObject.media || [])
-		.find((m: { iceUfrag: string; port: number }) => (
-			m.iceUfrag && m.port !== 0
+		.find((m: { port: number }) => (
```

Comment:
I think we should at least verify that the media section has `a=setup` attribute, right? Note that in lines below we return a value based on `mediaObject.setup` value.

Reply:
I'm gonna merge and then I'll add that check. Thanks!

Reply:
Done here: https://github.com/versatica/mediasoup-client/commit/eb90e61cbd1da810a5ac271fed02d650706802e6

Reply:
Oh... ok but it's not just about reverting it. We must also be ready to read a=setup from session level too. Could you please fill an issue? I'd do but I must go to Aste Nagusia soon.

Reply:
I've properly fixed it in this commit: https://github.com/versatica/mediasoup-client/commit/0381230b93abb1a0b296c21d1361037be552e870

Reply:
Released in 3.6.100